composer.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "fill-me-in/fill-me-in",
```

Comment:
vyplnit

Reply:
fixed, i kdyz nevim jesli to z te sablony nevyhodit, na nic to neni potreba

Reply:
Pravda.

Reply:
https://github.com/keboola/component-generator/issues/14